lowcodelanguage/build.gradle

```diff
@@ -0,0 +1,70 @@
+plugins {
+    id 'com.github.ben-manes.versions'
+    id 'org.javamodularity.moduleplugin' version '1.8.3' apply false
+    id 'org.cadixdev.licenser'
+}
+apply plugin: 'java-library'
+apply plugin: 'jacoco'
+apply plugin: 'org.javamodularity.moduleplugin'
+
+dependencyUpdates.rejectVersionIf { isNonStable(it.candidate.version) }
+
+dependencies {
+    compileOnly('org.jetbrains:annotations:21.0.1')
+    implementation(project(':fmlloader'))
+    implementation(project(':fmlcore'))
+}
+
+task sourcesJar(type: Jar) {
+    archiveClassifier = 'sources'
+    from sourceSets.main.allSource
+}
```

Comment:
This should instead use `java.withSourcesJar()`.

Reply:
This is how it is in javafmllanguage. Fix it in that place?